ui/src/pages/backoffice/Patron/PatronDetails/components/ItemsSearch/ItemsSearch.js

```diff
@@ -81,111 +64,79 @@ export default class ItemsSearch extends Component {
-          this.onKeyPressHandler(event.target);
-        }}
-        ref={input => {
-          this.searchInput = input;
-        }}
-      />
-    );
-  };
-
   renderResultsList = results => {
     return (
       <div className="results-list">
-        {this.renderHeader(results.hits.length)}
         <ItemsResultsList
           patronPid={this.props.patronDetails.user_pid}
           clearResults={this.clearResults}
           results={results}
-          checkoutItem={this.checkoutItem}
-          fetchPatronCurrentLoans={this.fetchUpdatedCurrentLoans}
+          clearSearchQuery={this.clearSearchQuery}
         />
       </div>
     );
   };
 
-  renderEmptyResults = (queryString, resetQuery) => {
+  clearSearchQuery = () => {
+    this.setState({ prevSearchQuery: '' });
+    this.clearResults();
+  };
+
+  renderSearchPrompt = () => {
     return (
       <Segment placeholder textAlign="center">
         <Header icon>
           <Icon name="search" />
-          Found no items matching this barcode.
+          No barcode provided.
         </Header>
         <div className="empty-results-current">
-          Current search phrase "{queryString}"
+          Type or paste the barcode to search for items
         </div>
-        <Segment.Inline>
-          <Button primary onClick={() => resetQuery('')}>
-            Clear query
-          </Button>
-        </Segment.Inline>
       </Segment>
     );
   };
 
-  renderHeader = totalResults => {
-    return <p>Found {totalResults} item(s).</p>;
-  };
-
   render() {
-    const { items, isLoading, error } = this.props;
-    const { prevSearchQuery } = this.state;
+    const { items, isLoading, error, queryString } = this.props;
     return (
-      <Segment className={'patron-items'}>
-        <Header as={'h3'}>Items</Header>
-        <Header.Subheader>Search items by barcode.</Header.Subheader>
-        <Container className={'search-bar'}>{this.renderSearchBar()}</Container>
+      <>
+        <Container className="search-bar spaced">
+          <SearchBar
+            action={{
+              icon: 'search',
+              onClick: () => this.executeSearch(),
+            }}
+            currentQueryString={queryString}
+            onInputChange={this.onInputChange}
+            executeSearch={this.executeSearch}
+            placeholder={'Search for physical copies...'}
+            onPaste={e => {
+              this.onPasteHandler(e);
+            }}
+          />
+        </Container>
         <Grid columns={1} stackable relaxed className="items-search-container">
           <Grid.Column width={16}>
             <Loader isLoading={isLoading}>
               <Error error={error}>
-                {!isEmpty(items.hits)
-                  ? this.renderResultsList(items)
-                  : this.renderEmptyResults(
-                      prevSearchQuery,
-                      this.updateQueryString
-                    )}
+                {queryString === '' || isEmpty(items)
```

Comment:
```suggestion
                {isEmpty(queryString) || isEmpty(items)
```

Reply:
nit: If I am not wrong we import `lodash` functions with `_` prefix.

Reply:
sure, I forgot about that conversation when I was writing this. Thanks for the reminder!